dockerfiles/base/scripts/base/startup_funcs.sh

```diff
@@ -403,12 +527,17 @@ start() {
   set -- "${@/\-\-debug/}"
   set -- "${@/\-\-offline/}"
   set -- "${@/\-\-trace/}"
+  set -- "${@/\-\-skip\:preflight/}"
+  set -- "${@/\-\-skip\:postflight/}"
+  set -- "${@/\-\-skip\:nightly/}"
+  set -- "${@/\-\-skip\:network/}"
+  set -- "${@/\-\-skip\:pull/}"
   
-    # The pre_init method is unique to each assembly. This method must be provided by 
+  # The cli_post_init method is unique to each assembly. This method must be provided by 
   # a custom CLI assembly in their container and can set global variables which are 
   # specific to that implementation of the CLI. This method must be called after
   # networking has been established and initial images downloaded.
-  cli_pre_init
+  cli_post_init
```

Comment:
I think we would be better suited to renaming the hooks to be `post_init` and `pre_init`.